### DISCUSSION TITLE
"id" : "OPT-201-Organization-TouchPoint

### DISCUSSION THREAD

**Discussion by rengit62** [OP-TED/eForms-SDK#161] [Q&A] [answered]
Hi !
in xsd cac:PartyIdentification is an element of TouchPointType
<xsd:complexType name="TouchPointType">
		<xsd:sequence>
**********************************************************************
			<xsd:element ref="cac:PartyIdentification" minOccurs="1" maxOccurs="1"/>
******************************************************************			
		</xsd:sequence>
	</xsd:complexType>

what do you think to set as its parentId ND-Touchpoint instead of ND-Organization ?
"id" : "OPT-201-Organization-TouchPoint",
    "parentNodeId" : "ND-Organization",
    "name" : "TouchPoint Technical Identifier",

Bye !
Renato

**Comment by bertrand-lorentz** [comment, marked as answer]
Thanks for the suggestion.
This is indeed one of the changes that we are doing to improve the structure defined in fields.json:
https://github.com/OP-TED/eForms-SDK/pull/162/commits/c29540c07dbe947ad5d8b29856ea95feaf90c577#diff-04b1c29add3adb0720b109b3acea38ea110fe7542d1e2786668bc7a8b16cac9cR27358

This should be merged into develop soon, and will be part of the next SDK version.

**Reply by rengit62** [reply]
Thanks to you !
another field to 'promote' to node ( in my opinion  of course ! )  is efac:Publication 
	<xsd:element name="Publication" type="PublicationType"/>
<xsd:complexType name="PublicationType">
		<xsd:sequence>
			<xsd:element ref="efbc:NoticePublicationID" minOccurs="1" maxOccurs="1"/>
			<xsd:element ref="efbc:GazetteID" minOccurs="1" maxOccurs="1"/>
			<xsd:element ref="efbc:PublicationDate" minOccurs="1" maxOccurs="1"/>
		</xsd:sequence>
	
in conceptual model

OPP-010-notice											efac:Publication/efbc:NoticePublicationID[@schemeName='ojs-notice-id']
OPP-011-notice											efac:Publication/efbc:GazetteID[@schemeName='ojs-id']
OPP-012-notice											efac:Publication/efbc:PublicationDate

( to avoid  ugly 'if' in the code )
Bye
Renato
